docs/addons/aws-load-balancer-controller.md

```diff
@@ -10,18 +10,23 @@ For more information about AWS Load Balancer Controller please see the [official
 ## Usage
 
 ```typescript
-import { AwsLoadBalancerControllerAddOn, ClusterAddOn, EksBlueprint }  from '@aws-quickstart/eks-blueprints';
-
-const addOn = new AwsLoadBalancerControllerAddon();
-const addOns: Array<ClusterAddOn> = [ addOn ];
+import 'source-map-support/register';
+import * as cdk from 'aws-cdk-lib';
+import * as blueprints from '@aws-quickstart/eks-blueprints';
 
 const app = new cdk.App();
-new EksBlueprint(app, 'my-stack-name', addOns, [], {
-  env: {
-      account: <AWS_ACCOUNT_ID>,
-      region: <AWS_REGION>
-  },
-});
+const account = <AWS_ACCOUNT_ID>;
+const region = <AWS_REGION>;
+const env: { account, region },
+
+const addOn = new blueprints.addons.AwsLoadBalancerControllerAddOn();
+const addOns: Array<blueprints.ClusterAddOn> = [ addOn ];
```

Comment:
this is not needed, let's drop this const. You can just pass addOn  to the builder:
```
.addOns(addOn)
```

Reply:
addressed.